guestbook/frontend-service.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     app: guestbook
     tier: frontend
 spec:
+  # comment or delete the following line if you want to use a LoadBalancer
+  type: NodePort 
```

Comment:
@cody-clark you can try, but I think we can't have both in the same YAML file. (I'm not sure it's a good practice to do so either.)

What would work in both platforms is to have 2 services (guestbook, guestbook-external) in a single file or separate files, but that would be more confusing. For now, let's keep it as is and I will update GKE tutorial.